<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zdeny_PhD_ShenanigansClass</class>
 <widget class="QMainWindow" name="Zdeny_PhD_ShenanigansClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>464</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Zdeny_PhD_Shenanigans</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>441</width>
      <height>421</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>157</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>157</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>96</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonClearLogs">
       <property name="text">
        <string>Clear Logs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAddLogs">
       <property name="text">
        <string>Log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close All Windows</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Zdeny's PhD Shenanigans</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>331</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Zdeny_s_PhD_Shenanigans"/>
   </widget>
   <widget class="QMenu" name="menuImage_Filtering">
    <property name="title">
     <string>Filtering</string>
    </property>
    <addaction name="actiongamma_contrast_brightness"/>
    <addaction name="actionfourier_filtering"/>
   </widget>
   <widget class="QMenu" name="menuOptimization">
    <property name="title">
     <string>Optimization</string>
    </property>
    <addaction name="actiontest_benchmark"/>
   </widget>
   <widget class="QMenu" name="menuImage_Registration">
    <property name="title">
     <string>IPC</string>
    </property>
    <addaction name="actionIPC_parameters"/>
    <addaction name="actionIPC_2pic_align"/>
    <addaction name="actionIPC_optimize"/>
    <addaction name="actionIPC_duel"/>
   </widget>
   <widget class="QMenu" name="menuNeural">
    <property name="title">
     <string>Neural</string>
    </property>
    <addaction name="actionANN"/>
    <addaction name="actionCNN"/>
   </widget>
   <widget class="QMenu" name="menuFractal">
    <property name="title">
     <string>Fractal</string>
    </property>
    <addaction name="actionMandelbrot"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionDebug"/>
    <addaction name="actionFITS_download_url_creator_2"/>
   </widget>
   <widget class="QMenu" name="menuAstrophysics">
    <property name="title">
     <string>Astrophysics</string>
    </property>
    <addaction name="actiondiffrot"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImage_Filtering"/>
   <addaction name="menuImage_Registration"/>
   <addaction name="menuAstrophysics"/>
   <addaction name="menuOptimization"/>
   <addaction name="menuNeural"/>
   <addaction name="menuFractal"/>
   <addaction name="menuTools"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionAbout_Zdeny_s_PhD_Shenanigans">
   <property name="text">
    <string>About Zdeny's PhD Shenanigans</string>
   </property>
  </action>
  <action name="actiongamma_contrast_brightness">
   <property name="text">
    <string>gamma / contrast / brightness</string>
   </property>
  </action>
  <action name="actionfourier_filtering">
   <property name="text">
    <string>fourier filtering</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionIPC_parameters">
   <property name="text">
    <string>IPC parameters</string>
   </property>
  </action>
  <action name="actionMandelbrot">
   <property name="text">
    <string>mandelbrot</string>
   </property>
  </action>
  <action name="actionANN">
   <property name="text">
    <string>ANN</string>
   </property>
  </action>
  <action name="actionCNN">
   <property name="text">
    <string>CNN</string>
   </property>
  </action>
  <action name="actiontest_benchmark">
   <property name="text">
    <string>test benchmark</string>
   </property>
  </action>
  <action name="actionIPC_optimize">
   <property name="text">
    <string>IPC optimize</string>
   </property>
  </action>
  <action name="actionIPC_duel">
   <property name="text">
    <string>2 IPC duel</string>
   </property>
  </action>
  <action name="actiondiffrot">
   <property name="text">
    <string>diffrot</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionFITS_download_url_creator">
   <property name="text">
    <string>FITS download url creator</string>
   </property>
  </action>
  <action name="actionFITS_download_url_creator_2">
   <property name="text">
    <string>FITS download url creator</string>
   </property>
  </action>
  <action name="actionIPC_2pic_align">
   <property name="text">
    <string>IPC 2 pic</string>
   </property>
  </action>
  <action name="actionFeature_matching">
   <property name="text">
    <string>feature matching</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Zdeny_PhD_Shenanigans.qrc"/>
 </resources>
 <connections/>
</ui>
